examples/ima.amp.html

```diff
@@ -0,0 +1,33 @@
+<!doctype html>
+<html ⚡>
+<head>
+    <meta charset="utf-8">
+    <title>IMA examples</title>
+    <link rel="canonical" href="amps.html" >
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+    <link href='https://fonts.googleapis.com/css?family=Questrial' rel='stylesheet' type='text/css'>
+    <link href="//vjs.zencdn.net/5.3/video-js.min.css" rel="stylesheet">
+    <link rel="stylesheet" href="http://gvabox.com/sbusolits/amp/css/videojs.ads.css" />
+    <link rel="stylesheet" href="http://gvabox.com/sbusolits/amp/css/videojs.ima.css" />
+    <script async custom-element="amp-ima" src="https://cdn.ampproject.org/v0/amp-ima-0.1.js"></script>
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+</head>
+<body>
+
+
+
+<h2>IMA</h2>
+
+<amp-ima width=640 height=360
+    layout="responsive"
+    data-width="640"
+    data-height="360"
+    data-src="http://rmcdn.2mdn.net/Demo/vast_inspector/android.mp4"
+    data-mime="video/mp4"
+    data-tag="https://pubads.g.doubleclick.net/gampad/ads?sz=640x480&iu=/124319096/external/single_ad_samples&ciu_szs=300x250&impl=s&gdfp_req=1&env=vp&output=vast&unviewed_position_start=1&cust_params=deployment%3Ddevsite%26sample_ct%3Dlinear&correlator=">
```

Comment:
why would we not want to design it so that you could wrap any amp-ad tag within the amp-ima tag rather than forcing them to give the raw url for  the adrequest.. also does it only work woth doubleclick what about any other ad server?